pkg/utils/expNotFoundPatch.go

```diff
@@ -1,13 +1,25 @@
 package utils
 
 import (
-	log "github.com/sirupsen/logrus"
+	//log "github.com/sirupsen/logrus"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+func (expStatus *ExperimentStatus) NotFoundExperimentStatus(experimentDetails *ExperimentDetails) {
```

Comment:
ExperimentStatus structures of various types (Initial, NotRunning, Awaiting, Completed) etc., can be initialized in a separate `status.go` file

Reply:
instead of `Awaiting` can it be `pending`? cc @ksatchit

Reply:
I think awaited would be suitable.